core/trino-main/src/main/java/io/trino/server/InternalCommunicationModule.java

```diff
@@ -28,6 +28,7 @@ protected void setup(Binder binder)
     {
         InternalCommunicationConfig internalCommunicationConfig = buildConfigObject(InternalCommunicationConfig.class);
         configBinder(binder).bindConfigGlobalDefaults(HttpClientConfig.class, config -> {
+            // Set defaults for all HttpClients in the same guice context
```

Comment:
This is not what I meant. I wanted to add a link `JwtAuthencatorSupportModule` and `Oauth2ServiceModule` here, so if someone changes defaults then should also consider updating these locations.

Reply:
Ok, please take yet another look then.